### PR TITLE
feat(Stories): support single page story

### PR DESCRIPTION
**Pull request by kseniya57**
Added single page stories support
Added possibility to provide custom controls
Added possibility to add custom styles for stories modal contents

### CODE REVIEW

**Comment by DarkGenius** [comment]
Seems like the button should be on the left
image

File: src/components/Stories/README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Component for displaying stories. It looks like a carousel in a modal with given
 | onPreviousClick     | `Function`      |          |         | Action when switching to previous story          |
 | onNextClick         | `Function`      |          |         | Action when switching to next story              |
 | disableOutsideClick | `Boolean`       |          | true    | If `true`, do not close stories on click outside |
+| className           | `string`        |          |         | Stories modal class                              |
+| controls            | `ReactNode`     |          |         | Custom action buttons                            |
```
[the remark this answers is not in the document]
Comment (kseniya57) [reply src/components/Stories/README.md:17]: Wee need to add some styles for the text

**Comment by DarkGenius** [comment]
> > Seems like the button should be on the left image
> 
> Why?

Because in case of multiple slides on the first slide the close button is on the left

File: src/components/Stories/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DarkGenius) [reply src/components/Stories/README.md:17]: I mean the ability to render custom controls

[the remark this answers is not in the document]
Comment (kseniya57) [reply src/components/Stories/README.md:17]: We have a button for the transition to another page

[the remark this answers is not in the document]
Comment (DarkGenius) [reply src/components/Stories/README.md:17]: The general recommendation is that it is undesirable to add the possibility of extensive customization by rendering arbitrary elements, it is better to go by enriching the component with the missing functionality. Let's think about this, shall we?

[the remark this answers is not in the document]
Comment (kseniya57) [reply src/components/Stories/README.md]: Added `actions` property, which is an array of button props

File: src/components/Stories/__stories__/Stories.stories.tsx [hunk 2]
```diff
@@ -76,3 +76,21 @@ Default.argTypes = {
...
+};
+
+export const WithCustomActions = DefaultTemplate.bind({});
+WithCustomActions.args = {
+    open: false,
+    items: [items[0]],
+    actions: [
+        {
+            view: 'action',
+            children: 'Перейти',
```
Comment (DarkGenius) [inline src/components/Stories/__stories__/Stories.stories.tsx:93]: English please)

File: src/components/Stories/components/StoriesLayout/StoriesLayout.tsx [hunk 3]
```diff
@@ -67,21 +71,35 @@ export const StoriesLayout = (props: StoriesLayoutProps) => {
...
+                                    onClick={props.handleGotoNext}
+                                    view="action"
+                                    size="l"
+                                    width="max"
+                                >
                                     {i18n('label_next')}
                                 </Button>
                             )}
+                            {(!hasSteps || IndexType.End === props.indexType) &&
+                                props.actions?.map((actionConfig, index) => (
```
Comment (DarkGenius) [inline src/components/Stories/components/StoriesLayout/StoriesLayout.tsx:100]: Will the layout break if there are multiple actions?

**Review by DarkGenius — APPROVED** [review]

**Review by amje — APPROVED** [review]

**Review by ogonkov — APPROVED** [review]

**Pull request merged**